rector.yaml

```diff
@@ -23,3 +23,53 @@ parameters:
 
     # so Rector code is still PHP 7.2 compatible
     php_version_features: '7.2'
+
+    rector_recipe:
+        package: "PHPUnit"
+        name: "CreateMockToCreateStub"
+        node_types:
+            - "MethodCall"
+        description: "Replaces createMock() with createStub() when relevant"
+        code_before: >
+            <?php
+
+            use PHPUnit\Framework\TestCase;
+
+            class MyTest extends TestCase
+            {
+                public function testItBehavesAsExpected(): void
+                {
+                    $stub = $this->createMock(\Exception::class);
+                    $stub->method('getMessage')
+                        ->willReturn('a message');
+                    $mock = $this->createMock(\Exception::class);
+                    $mock->expects($this->once())
+                        ->method('getMessage')
+                        ->willReturn('a message');
+                    self::assertSame('a message', $stub->getMessage());
+                    self::assertSame('a message', $mock->getMessage());
+                }
+            }
+        code_after: >
+            <?php
+
+            use PHPUnit\Framework\TestCase;
+
+            class MyTest extends TestCase
+            {
+                public function testItBehavesAsExpected(): void
+                {
+                    $stub = $this->createStub(\Exception::class);
+                    $stub->method('getMessage')
+                        ->willReturn('a message');
+                    $mock = $this->createMock(\Exception::class);
+                    $mock->expects($this->once())
+                        ->method('getMessage')
+                        ->willReturn('a message');
+                    self::assertSame('a message', $stub->getMessage());
+                    self::assertSame('a message', $mock->getMessage());
+                }
+            }
+        source:
+            - https://github.com/sebastianbergmann/phpunit/issues/3120
+        set: "phpunit90"
```

Comment:
I will revert the commit that contains this file, it's just here for reference.

Reply:
Tip: you can put this into `create-rector.yaml` file and it won't be commited.
https://github.com/rectorphp/rector/blob/master/.gitignore#L12

That's how I use it